### PR TITLE
adds space funeral mobs

### PR DESCRIPTION
**Pull request by Blute1**
added the space funeral mobs we all know and love

:cl: Spacedong
add: Added space funeral mobs, they are currently adminspawn or gold core spawnable.
/:cl:

### CODE REVIEW

**Comment by dreamcrab** [comment]
i thought i said yes to this a million years ago

**Comment by zamolxius** [comment]
ADD

**Comment by Dudeinyourbackyard** [comment]
+1 again because the last one DIED

**Comment by Pyko1** [comment]
isnt +2 enough again?

**Comment by GuyonBroadway** [comment]
Tis

**Comment by dreamcrab** [comment]
+999999999999999999999

File: hippiestation/code/modules/mob/living/simple_animal/hostile/space_funeral.dm [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	melee_damage_lower = 1
+	melee_damage_upper = 2
+	attacktext = "bites"
+	attack_sound = 'sound/weapons/bite.ogg'
+	gold_core_spawnable = HOSTILE_SPAWN
+
+	proc/splat()
+		src.health = 0
+		src.stat = DEAD
+		src.icon_state = "bloodflea-dead"
```
Comment (praisenarsie) [inline hippiestation/code/modules/mob/living/simple_animal/hostile/space_funeral.dm:65]: wot in fucking tarnation is this @Blute1

File: hippiestation/code/modules/mob/living/simple_animal/hostile/space_funeral.dm [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+/mob/living/simple_animal/hostile/space_funeral/leghorse
+	name = "leghorse"
+	desc = "A sanity-destroying otherthing. It seems happy"
```
Comment (praisenarsie) [inline hippiestation/code/modules/mob/living/simple_animal/hostile/space_funeral.dm]: . at the end

File: hippiestation/code/modules/mob/living/simple_animal/hostile/space_funeral.dm [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+	maxHealth = 50
+	melee_damage_lower = 8
+	melee_damage_upper = 10
+	attacktext = "bites"
+	attack_sound = 'sound/weapons/bite.ogg'
+	gold_core_spawnable = HOSTILE_SPAWN
+
+/mob/living/simple_animal/hostile/space_funeral/bloodflea
+	name = "bloodflea"
+	desc = "A small sanity-destroying otherthing. It seems weak"
```
Comment (praisenarsie) [inline hippiestation/code/modules/mob/living/simple_animal/hostile/space_funeral.dm]: . at the end

File: hippiestation/code/modules/mob/living/simple_animal/hostile/space_funeral.dm [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+	melee_damage_lower = 1
+	melee_damage_upper = 2
+	attacktext = "pokes"
+	attack_sound = 'sound/weapons/bite.ogg'
+	faction = list("funeral")
+
+
+/mob/living/simple_animal/hostile/space_funeral/philip
+	name = "crier"
+	desc = "A sanity-destroying otherthing. It seems sad"
```
Comment (praisenarsie) [inline hippiestation/code/modules/mob/living/simple_animal/hostile/space_funeral.dm]: . at the end

**Comment by GuyonBroadway** [comment]
Waaaaait this has a +2 and a sucessful code check.

**Comment by GuyonBroadway** [comment]
Actually nevermind, no it don't

**Comment by Carbonhell** [comment]
What is this code
>Relative pathing
Jesus christ fix it

**Comment by Carbonhell** [comment]
Is this PR stale?It needs to be fixed or it ain't going in
@Blute1 ???

**Comment by Dudeinyourbackyard** [comment]
:spenge:

**Comment by Pyko1** [comment]
anyone interested in fixing?

Anyone with write access should be able to fix online.
Will prolly close in few days otherwise.

**Comment by praisenarsie** [comment]
how about i close it now

**Pull request closed without merging**